Resources/Prototypes/Loadouts/neck.yml

```diff
@@ -102,3 +142,232 @@
   exclusive: true
   items:
     - ClothingNeckTransPin
+
+- type: loadout
+  id: LoadoutItemsAutismPin
+  category: Neck
+  cost: 1
+  exclusive: true
+  items:
+    - ClothingNeckAutismPin
+
+- type: loadout
+  id: LoadoutItemsGoldAutismPin
+  category: Neck
+  cost: 1
+  exclusive: true
+  items:
+    - ClothingNeckGoldAutismPin
+
+# Bedsheets
+- type: loadout
+  id: LoadoutNeckBedsheetBlack
+  category: Neck
+  cost: 2
+  exclusive: true
+  items:
+    - BedsheetBlack
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Civilian
+    - !type:CharacterDepartmentRequirement
+       inverted: true
+       departments:
+         - Command
+
+- type: loadout
+  id: LoadoutNeckBedsheetBlue
+  category: Neck
+  cost: 2
+  exclusive: true
+  items:
+    - BedsheetBlue
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Civilian
+    - !type:CharacterDepartmentRequirement
+       inverted: true
+       departments:
+         - Command
```

Comment:
What's the thought process?

Reply:
We were discussing the need to make Command roles recognizable as their role, and bedsheets seem to make it pretty difficult to recognize someone's role. This is more to prevent the HoP (part of the Civilian department) from wearing bedsheets that only non-head civilians are supposed to wear.

Reply:
Unprofessional? It's a bed sheet

Though, imo loadout items are things you keep with you and wear from shift to shift. A bed sheet feels odd for a worker to wear, even if they're a bartender or something

Reply:
We want to specifically restrict the available color pallets of Command and Security so as to prevent them from rendering themselves unrecognizable with their loadouts, which as much as we like people having ridiculous freedom of character customization, there still has to be some concessions to gameplay. In practice it means that for instance Security can generally wear any combination of Red/Black/Dark Grey/some Blues, and still be immediately recognizable as Sec. The biggest offenders in the "Job Recognition" issue are Uniform and Neck.

We previously had the full list of PrideVend cloaks available in loadouts, and had to remove them because at the time Department Restrictions weren't in the game yet.